src/components/Footer.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+const Footer = () => {
+  return (
+    <div className='footer'>
+    <div>©2023 uproar</div>
```

Comment:
All the divs that are nested inside the`<div className='footer'>` should be indented

Reply:
Adjusted in latest commit to test-coverage.